dao/schema.graphql

```diff
@@ -38,6 +38,7 @@ type Resolution @entity {
   executionTo: [Bytes!]!
   executionData: [Bytes!]!
   executionTimestamp: BigInt
+  addressedContributor: Bytes!
```

Comment:
@vrde can you please either remove the bang here or set something in place of null in the resolution created handler? That's the problem preventing the indexing

Reply:
IMO the issue is somewhere here:

`Apr 22 07:51:51.447 ERRO Handler skipped due to execution failure, error: Mapping aborted at generated/schema.ts, line 877, column 7, with message: Cannot return null for a required field.    wasm backtrace:     0: 0x4e70 - <unknown>!src
/resolution-manager-mapping/handleResolutionTypeCreated , handler: handleResolutionTypeCreated, sgd: 2, subgraph_id: Qmceo8Tw9hfa3iGdEf8BubZZH7brUXSQL55q4fuwUfW21K, component: SubgraphInstanceManager`

https://github.com/NeokingdomDAO/subgraph/blob/31921b9b6f4940376a2096f478b4b0a3f45d61be/dao/generated/schema.ts#L874-L881

Reply:
Ah yes, it's on another place the problem. Resolving this